### ISSUE TITLE
create the NVDLA  IP for the nv_small with Vivado and  manage the connections with the Zynq UltraScale

### ISSUE THREAD

**Issue by ghost** [VVViy/VVViy.github.io#9]
did you try to create the NVDLA  IP for the nv_small with Vivado? If so, how did you manage the connections with the Zynq UltraScale+?

**Comment by VVViy** [comment]
Yeah, i did it and have done. In terms of "how", if you know chinese，you can refer to my blog in this repo [nv_small FPGA Mapping Workflow](https://vvviy.github.io/2018/09/12/nv_small-FPGA-Mapping-Workflow-I/) .

**Comment by VVViy** [comment]
Yeah, i did it and have done. In terms of "how", if you know chinese，you can refer to my blog in this repo [nv_small FPGA Mapping Workflow](https://vvviy.github.io/2018/09/12/nv_small-FPGA-Mapping-Workflow-I/) .

**Comment by honorpeter** [comment]
Thank you very much. I will try it as your suggested.

**Comment by VVViy** [comment]
It's cool, good luck.

**Comment by ghost** [comment]
I am running nvdla_small spec on Xiliinx Ultrascale Board ZCU102
 But,There's a problem : NVDLA a0000000.nvdla_small: failed to register drm device。
cat /proc/interrupts    is also not right。
Why,how to solve it? who have the same problem?
